src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslX509CertificateReader.cs

```diff
@@ -411,6 +410,61 @@ public X500DistinguishedName IssuerName
             }
         }
 
+        public bool GetPolicyData(out byte[] ApplicationCertPoliciesData, out byte[] CertPoliciesData, out byte[] CertPolicyMappingsData,
+                           out byte[] CertPolicyConstraintsData, out byte[] EnhancedKeyUsageData, out byte[] InhibitAnyPolicyExtensionData)
+        {
+
+            ApplicationCertPoliciesData = null;
+            CertPoliciesData = null;
+            CertPolicyMappingsData = null;
+            CertPolicyConstraintsData = null;
+            EnhancedKeyUsageData = null;
+            InhibitAnyPolicyExtensionData = null;
+
+            int extensionCount = Interop.Crypto.X509GetExtCount(_cert);
+
+            for (int i = 0; i < extensionCount; i++)
+            {
+                IntPtr ext = Interop.Crypto.X509GetExt(_cert, i);
+
```

Comment:
Nit: unnecessary blank line (same in the other cases below

Reply:
I removed some blanks leaving cluster of related parts without blank lines.